scripts/base.py

```diff
@@ -0,0 +1,80 @@
+import os
+import subprocess
+import sys
+import re
+
+ROOT = (
+    subprocess.check_output(["git", "rev-parse", "--show-toplevel"])
+    .decode("ascii")
+    .strip()
+)
+PROJECT_NAME = "Strawberry"
+
+# Files configuration
+
+RELEASE_FILE_NAME = "RELEASE.md"
+RELEASE_FILE = os.path.join(ROOT, RELEASE_FILE_NAME)
+
+PROJECT_TOML_FILE_NAME = "pyproject.toml"
+PROJECT_TOML_FILE = os.path.join(ROOT, PROJECT_TOML_FILE_NAME)
+
+CHANGELOG_FILE_NAME = "CHANGELOG.md"
+CHANGELOG_FILE = os.path.join(ROOT, CHANGELOG_FILE_NAME)
+
+# Git configuration
+
+GIT_USERNAME = "USERNAME_HERE"
+GIT_EMAIL = "EMAIL_HERE"
```

Comment:
do we need environment variables for these?

Reply:
It depends, I think we can keep those in the code as they will be public anyway?

Reply:
good point :)